README.md

```diff
@@ -6,56 +6,69 @@
 
-After some time, a lot of questions and answers will be added to rocket fuel. You can search for questions in rocket fuel as well as answer questions there. To filter out all the noise, a rating system is used for questions and answers, so that the most relevant questions and answers will be shown to the user.
+After some time, a lot of questions and answers will be added to Rocket fuel. You can search for questions in Rocket fuel as well as answer questions there. To filter out all the noise, a rating system is used for questions and answers, so that the most relevant questions and answers will be shown to the user.
 
 ## This is how it works. 
 
-### The slack way
+### The Slack way
 
-A thread is started in slack, the rocket fuel bot detects that a question has been asked and the slack rocket fuel bot asks the members of the thread if they want to save the question and the answers in rocket fuel, so that anyone in the organisation easily can find the question and the answers later on, instead of asking the question again. 
+As a thread is started in Slack, the Rocket fuel bot detects that a question has been asked and the bot asks the members of the thread if they want to save the question and the answers in Rocket fuel, so that anyone in the organisation easily can find the question and the answers later on, instead of asking the question again. 
 
 ### The normal way
-A user can add a question to rocket fuel directly in the ui, and then wait for anyone to answer it, this is very similar to how it works on stack overflow. 
+A user can add a question to Rocket fuel directly in the UI, and then wait for anyone to answer it, this is very similar to how it works on Stack overflow. When an answer has been submitted, the user will be notified through Slack. The user will be given a link to the question. Then its up to the user to accept the answer or write back. 
 
 
 ### Contribution
 
-Feel free to contribute to rocket fuel. Create a issue, when the issue is accepted, create a fork and create a pull request. 
+Feel free to contribute to Rocket fuel by initially creating issues on Github. When the issue has been accepted then submit your changes in a pull request from a forked repository. 
 
 #### Prerequisites
 
-* Authentication requires you to set up a client in the Google Developer console - javascript origin and redirect URI should be set to where rocket fuel is hosted, such as http://localhost:8083.
-* Postgres - run one in docker with
-```
-docker run --name rocketfuel-postgres -p 15432:5432 -e POSTGRES_PASSWORD=mysecretpassword -e POSTGRES_DB=rocket-fuel -d postgres
-```
-* A [slack bot](https://api.slack.com/apps)
+To be able to run and develop Rocket fuel, you will need to have the following installed on your machine.
 
-#### Running in intellij
+* Java 11
+* Maven 3.04 or higher
```

Comment:
Why 3.04? It's from 2012 :D

Reply:
just the limitation of some of our plugins that require 3.04 or higher. somehow i got an older version on one of my laptops that took me some time solve :) . It's in the release process that we need 3.04 or higher. The one that is "dead and buried" :) But can change to some never version if you like :) but it works on that one :)